docs/1_installation.md

```diff
@@ -2,15 +2,18 @@
 
 The instructions for installation are available in our documentation, [accessible here.](https://scilifelabdatacentre.github.io/dds_cli/#install-the-command-line-interface-cli-dds-cli)
 
-They differ slightly between Mac or Windows so, follow the section you need. Once installed, verify it by running:
+They differ slightly depending on your operating system, so follow the section you need (MacOS, Windows or Linux distributions). Once installed, verify it by running:
```

Comment:
Maybe I should change this to say Ubuntu instead of Linux dists?

Reply:
Is it more general to say Linux. I guess if they use Ubuntu, they know enough to know this

Reply:
I guess in terms of what we aim to support (and have tested): is it all standard Linux distributions or are we more targeting Ubuntu and if people get it to work on other things good for them?

Reply:
> Is it more general to say Linux. I guess if they use Ubuntu, they know enough to know this

True.


> I guess in terms of what we aim to support (and have tested): is it all standard Linux distributions or are we more targeting Ubuntu and if people get it to work on other things good for them?

I **think** it should work on more dists than Ubuntu when installing via PyPI, but I'm not 100% sure. Regarding the executables, it's only Ubuntu.